### DISCUSSION TITLE
App Store "Revoke" notification

### DISCUSSION THREAD

**Discussion by kiritokatklian** [imdhemy/laravel-in-app-purchases#40] [Ideas]
Hello @imdhemy. First of all great work on this package!

So I have come across a new notification type from the App Store which doesn't seem to be handled by this package.
This is how Apple describes it.
```
REVOKE

Indicates that an in-app purchase the user was entitled to through Family Sharing is no longer available through sharing.
```

It would be nice to support this in an upcoming update.

**Comment by imdhemy** [comment]
@kiritokatklian 
Great, I'm gonna implement it ASAP. Thank you!
